### DISCUSSION TITLE
Can you tell me what it means "Are you willing to submit a PR?" in issues report?

### DISCUSSION THREAD

**Discussion by wmurw** [msys2/MINGW-packages#21778] [Q&A] [answered]
Can you tell me what it means "Are you willing to submit a PR?" in issues report?

**Comment by mmuetzel** [comment, marked as answer]
That question asks whether you'd be willing to invest some of your own time to track down a solution for the issue that you are describing and to open a PR (pull request) with that solution so the whole community gains from it.
